src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -1948,8 +1948,8 @@ internal bool Run(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader
                 stateObj._syncOverAsync = true;
 
                 bool dataReady;
-                bool result = TryRun(runBehavior, cmdHandler, dataStream, bulkCopyHandler, stateObj, out dataReady);
-                Debug.Assert(result == true, "Should never return false when _syncOverAsync is set");
+                TdsOperationStatus result = TryRun(runBehavior, cmdHandler, dataStream, bulkCopyHandler, stateObj, out dataReady);
+                Debug.Assert(result == TdsOperationStatus.Done, "Should never return false when _syncOverAsync is set");
```

Comment:
the message probably needs to be updated, there is no `false` anymore

Reply:
Done.